src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1,23 +1,221 @@
 package is.hail.io.vcf
```

Comment:
These changes seem to just use `RegionValueBuilder`. Is this an orthogonal set of changes we could make a separate PR?